apps/site/assets/ts/stop/components/amenities/FareSalesAmenityCard.tsx

```diff
@@ -63,12 +63,20 @@ const fareTable = (tableData: [string, string][]): React.ReactElement => (
         </tr>
       </thead>
       <tbody>
-        {tableData.map(([name, price]) => (
-          <tr key={name}>
-            <td>{name}</td>
-            <td>{price}</td>
-          </tr>
-        ))}
+        {tableData.map(([name, price]) => {
+          // Backend lowercases every word after the first word in `name`.  This fixes the one
+          // edge case for Commuter Rail
+          const changedName = name.replace(
+            /Commuter rail/g,
+            () => "Commuter Rail"
+          );
```

Comment:
**suggestion:** That particular backend function is only used for populating this table... so just change this in the backend.

Reply:
The cause of this problem is on the backend.  There is a [capitalize function](https://github.com/mbta/dotcom/blob/master/apps/site/lib/site_web/controllers/fare_controller.ex#L102) that causes it to be the bad case.  I will look into moving this back there but just wanted to clarify.